python/lsst/verify/compatibility/metricRegistry.py

```diff
@@ -0,0 +1,206 @@
+# This file is part of verify.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["register", "registerMultiple"]
+
+from lsst.pex.config import Config, ConfigDictField, Registry
+from .metricTask import MetricTask
+
+
+def register(name):
+    """A class decorator that registers a `lsst.verify.compatibility.MetricTask`
+    with a central repository.
+
+    Parameters
+    ----------
+    name : `str`
+        The name under which this decorator will register the
+        `~lsst.verify.compatibility.MetricTask`.
+
+    Raises
+    ------
+    RuntimeError
+        Raised if another class has already been registered under ``name``.
+    ValueError
+        Raised if this decorator is applied to a class that is not a
+        `lsst.verify.compatibility.MetricTask`.
+
+    Notes
+    -----
+    This decorator must be used for any `~lsst.verify.compatibility.MetricTask`
+    that is to be used with `lsst.verify.compatibility.MetricsControllerTask`.
+    If a task will *only* be run as a `~lsst.pipe.base.PipelineTask`, it should
+    not have this decorator.
```

Comment:
I think this was me being a bit too pedantic. `lsst.verify.compatibility.MetricTask` does not depend on `MetricsControllerTask`, so *in theory* it could be used without it (e.g., as a subtask of the task it's instrumenting). However, I'm not actually doing anything to support that usage.

If I just remove the second sentence, will that make more sense?

Reply:
Removing the second sentence will at least make it less confusing.